test/e2e/systemd_activate_test.go

```diff
@@ -64,12 +65,23 @@ var _ = Describe("Systemd activate", func() {
 		Expect(err).ToNot(HaveOccurred())
 		addr := net.JoinHostPort(host, strconv.Itoa(port))
 
+		// Make a temporary root directory
+		rootDir := filepath.Join(tempDir, "server_root")
+		err = os.Mkdir(rootDir, 0755)
+		Expect(err).ToNot(HaveOccurred())
+		defer os.RemoveAll(rootDir)
+
+		if selinux.GetEnabled() {
+			label := testUtils.SystemExec("chcon", []string{"-t", "container_var_lib_t", rootDir})
```

Comment:
This is not correct for rootless. Are we only checking rootful.

Rather then use container_var_lib_t, it would be better to use either
chcon --reference /var/lib/containers rootDir

But this could cause problems where we don't have podman installed.  Do we ever do this?

Reply:
Thank you for your review. I fixed it.
I have verified that both root and rootless are successed even if SELinux is enabled.

> But this could cause problems where we don't have podman installed.

If a directory such as `/var/lib/containers` doesn't exist (podman isn't installed),
chcon returns an error with `No such file or directory`.

As a result, e2e test will be failed in the following line:

https://github.com/containers/podman/blob/06e9cf6846ddb2510c78d9978e569914ae3a9665/test/e2e/systemd_activate_test.go#L83